docs: expand pattern export and add note for Windows users (#131)
Inbestigator/dressed: www/content/components.md

## Dynamic Component IDs

> [!IMPORTANT]

> > If you are are Windows you will encounter issues with the file name method for dynamic component IDs. It is recommended to use [Pattern Export](#pattern-export) over filenames in that instance.

Reviewer: ```suggestion
> If you are on Windows you will encounter issues with dynamic component IDs in file names. It is recommended to export a [pattern variable](#pattern-export) instead.
```

If you'd like to add some regex syntax, you can simply do `(...)`. This can be paired with an argument to ensure the argument value is correct.

### Pattern Export

If you have a complex dynamic ID or you're on Windows you may opt to use the `pattern` export over file name. Using pattern export will override file name completely.

#### Before

```sh

> /trivia_guess_:answer.ts

Reviewer: ```suggestion
.../trivia_guess_:answer.ts
```
or
```suggestion
trivia_guess_:answer.ts
```
Could remove confusion about it possibly being a slash command.
The second is also more consistent with the filename comment in the after section
